cmd/plugins_test.go

```diff
@@ -313,3 +361,25 @@ func TestLambdaFileWalker(t *testing.T) {
 		}
 	})
 }
+
+func signalHandler(tempRootDir string, cleanup func()) {
+	signalChan := make(chan os.Signal, 1)
+	signal.Notify(signalChan, syscall.SIGINT, syscall.SIGTERM)
+	exitChan := make(chan int)
+	go func() {
+		s := <-signalChan
+		test.FuncEnter()
+		defer test.FuncExit()
+		switch s {
+
+		case syscall.SIGINT, syscall.SIGTERM, syscall.SIGQUIT:
+			fmt.Println("Received an ", s.String(), "cleaning ", tempRootDir, "and exiting")
+			cleanup()
+			os.Exit(0)
```

Comment:
The exit code should be non-zero (1 is fine.)

Reply:
Okay